test/packagescanner.go

```diff
@@ -104,7 +104,7 @@ func (tc ScannerTestcase) RunSubset(ctx context.Context, n int) func(*testing.T)
 			}
 
 			if !cmp.Equal(p, g) {
-				t.Error(cmp.Diff(p, g))
+				t.Errorf("%v-%v-%v: %v", p.Name, p.RepositoryHint, p.Filepath, cmp.Diff(p, g))
```

Comment:
This helped a lot in cases where the package names matched. I was going to delete this, but it may be helpful again someday